cpp/src/slice2js/JsUtil.h

```diff
@@ -6,30 +6,26 @@
 #include "../Ice/OutputUtil.h"
 #include "../Slice/Parser.h"
 
-namespace Slice
+namespace Slice::JavaScript
 {
```

Comment:
I added `[[nodiscard]]`, removed `static`, and added names to any parameters which were missing them.
These are the only changes I made to these methods.